plugins/agent_types/AcquiaLiftAgent.inc

```diff
@@ -2248,47 +2319,49 @@ class AcquiaLiftReport extends AcquiaLiftReportBase {
-      $targeting_data = $report_data['targeting'][$feature_string];
+        // Get the data from the targeting report for this feature.
+        $targeting_data = $report_data['targeting'][$feature_string];
 
-      // Create a hash of choice numbers for stability report.
-      // Don't show system-defined features.
-      if ($targeting_data['system'] === TRUE) {
-        continue;
-      }
-      foreach($feature as $choice_id => $choice) {
-        $report_data['context']['features'][$feature_string][$choice_id] = array(
-          'counter' => $choice['counter'],
-          'choice_id' => $choice['choice_id'],
-          'best' => ($isAdaptive && $targeting_data['favored_selection'] === $choice['raw_label']),
-          'decisions' => $choice['decisions'],
-          'lift_default' => $choice['control'] ? self::DATA_NA : $choice['lift_default'],
-          'lift_default_positive' => $choice['unformatted']['lift_default'] > self::LIFT_THRESHHOLD,
-          'lift_random' => $choice['lift_random'],
-          'lift_random_positive' => $choice['unformatted']['lift_random'] > self::LIFT_THRESHHOLD,
-          'control' => $choice['control'],
-          'feature_label' => $feature_label,
-          'goals' => $choice['goals'],
-          'conversion' => $choice['conversion'],
-        );
-        $option_numbers[$feature_string . '|' . $choice['raw_label']] = $choice['counter'];
+        // Create a hash of choice numbers for stability report.
+        // Don't show system-defined features.
+        if ($targeting_data['system'] === TRUE) {
+          continue;
+        }
+        foreach ($feature as $choice_id => $choice) {
+          $report_data['context']['features'][$feature_string][$choice_id] = array(
+            'counter' => $choice['counter'],
+            'choice_id' => $choice['choice_id'],
+            'best' => ($isAdaptive && $targeting_data['favored_selection'] === $choice['raw_label']),
+            'decisions' => $choice['decisions'],
+            'lift_default' => $choice['control'] ? self::DATA_NA : $choice['lift_default'],
+            'lift_default_positive' => $choice['unformatted']['lift_default'] > self::LIFT_THRESHHOLD,
+            'lift_random' => $choice['lift_random'],
+            'lift_random_positive' => $choice['unformatted']['lift_random'] > self::LIFT_THRESHHOLD,
+            'control' => $choice['control'],
+            'feature_label' => $feature_label,
+            'goals' => $choice['goals'],
+            'conversion' => $choice['conversion'],
+          );
+          $option_numbers[$feature_string . '|' . $choice['raw_label']] = $choice['counter'];
+        }
       }
     }
 
     // Build the experiment report data
-    $report_data['experiment']['choices'] = $report_data['confidence']['features'][self::NO_FEATURES];
+    $report_data['experiment']['choices'] = isset($report_data['confidence']['features'][self::NO_FEATURES]) ? $report_data['confidence']['features'] : array();
```

Comment:
Just checking that this is what's intended - should it not be
`isset($report_data['confidence']['features'][self::NO_FEATURES]) ? $report_data['confidence']['features'][self::NO_FEATURES] : array();`
?